cli/src/pcluster/schemas/cluster_schema.py

```diff
@@ -1236,7 +1236,9 @@ class SchedulerPluginRequirementsSchema(BaseSchema):
     supports_cluster_update = fields.Bool(metadata={"update_policy": UpdatePolicy.UNSUPPORTED})
     supported_parallel_cluster_versions = fields.Str(
         metadata={"update_policy": UpdatePolicy.UNSUPPORTED},
-        validate=validate.Regexp(r"^((>|<|>=|<=)?[0-9]+\.[0-9]+\.[0-9]+,\s*)*(>|<|>=|<=)?[0-9]+\.[0-9]+\.[0-9]+$"),
+        validate=validate.Regexp(
+            r"^((>|<|>=|<=)?[0-9]+\.[0-9]+\.[0-9]+[a-z]?[0-9]?,\s*)*(>|<|>=|<=)?[0-9]+\.[0-9]+\.[0-9]+[a-z]?[0-9]?$"
```

Comment:
probably it's even more correct to have `([a-z][0-9]+)*`

Reply:
Changed. With the change we also support version like 3.1.0a111b12

Reply:
ah right it should be `?` not `*`

`^((>|<|>=|<=)?[0-9]+\.[0-9]+\.[0-9]+([a-z][0-9]+)?,\s*)*(>|<|>=|<=)?[0-9]+\.[0-9]+\.[0-9]+([a-z][0-9]+)?$`

Reply:
Changed